### ISSUE TITLE
Tire pressure "UNSPECIFIED" since 3.3.16

### ISSUE THREAD

**Issue by Tuxie** [Dielee/volvo2mqtt#287]
**Describe the bug**
Since the 3.3.16 Volvo System update on my XC60, the tire pressure levels are reported as "UNSPECIFIED" instead of "NO_WARNING". 

**Screenshots**:<br>
<img width="323" alt="Screenshot 2024-12-12 at 14 37 39" src="https://github.com/user-attachments/assets/79109e83-3ee3-47c0-b126-24cc4044cd28" />

**Version info**:<br>
volvo2mqtt v1.10.5
Home Assistant 2024.12.2
Volvo System Software v3.3.16

**Comment by akamming** [comment]
This is mijn from this integration. The integration just reports what the Volvo api reports

and the Volvo api now report unspecified.

so your issue should be with Volvo

**Comment by Vinisz** [comment]
I have the same. I agree it's with Volvo, anyone any idea where I can report this at Volvo ?

**Comment by akamming** [comment]
There is a contact link on the bottom of [this page](https://developer.volvocars.com/apis/)

**Comment by Vinisz** [comment]
Spotted, thanks... I dropped them an e-mail, will share results here

**Comment by AlexZuiderwijk** [comment]
In the readme from software update 3.3.16 it is mentioned you have to store
the tire pressure reference values again in the car..... this solved it for
me
Alex

Op za 14 dec 2024 07:52 schreef Vinisz ***@***.***>:

**Comment by ivanfmartinez** [comment]
For reference here after upgrade to 3.3.16 we have stored tire pressure like mentioned by @AlexZuiderwijk and we still get UNSPECIFIED for tire sensor. 

I agree that is volvo server related not volvo2mqtt

**Comment by Elekesey** [comment]
Same behavior to me : Still have UNSPECIFIED after stored tire pressure following software update.
@Vinisz did you received any feedback from Volvo ?

**Comment by Vinisz** [comment]
No, nothing, no reply from Volvo, I checked the car also and "reset" the tyre pressures etc, no change.
I just sent Volvo an e-mail again.

**Comment by Elekesey** [comment]
Thanks for your feedback.

I just sent an email to Volvo via their developpers portal. Will keep you informed.

Le 28 déc. 2024 à 13:06, Vinisz ***@***.***> a écrit :

﻿

No, nothing, no reply from Volvo, I checked the car also and "reset" the tyre pressures etc, no change.
I just sent Volvo an e-mail again.
